### DISCUSSION TITLE
Set Node position to X,Y coordinate

### DISCUSSION THREAD

**Discussion by KSilverman** [reaviz/reaflow#121] [Q&A]
Is there any way to set a node to a X and Y coordinate? I tried putting the x and y properties in the Node Props, but it doesn't seem to be budging. Basically in essence I want to be able to place nodes relative to other positions in a parent div. Is this already a feature that can be implemented? If there is, does it include any fancy CSS to make the boundaries relative to other positions of elements on the DOM?

I don't know if anyone else is running into this issue, and I've been combing through the documentation to try to figure this out. Please let me know! Thanks!

**Comment by amcdnl** [comment]
Ya - XY is not supported because of ELKJS - See issue here: https://github.com/reaviz/reaflow/issues/4
